tailcfg/tailcfg.go

```diff
@@ -532,6 +659,9 @@ func (ni *NetInfo) portMapSummary() string {
 	return prefix + conciseOptBool(ni.UPnP, "U") + conciseOptBool(ni.PMP, "M") + conciseOptBool(ni.PCP, "C")
 }
 
+// View returns a read-only accessor for the NetInfo object.
```

Comment:
Go doesn't really ever use the word "object". Maybe "a read-only accessor for ni." ?